modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc

```diff
@@ -96,84 +96,100 @@ a|
 
 |Disabled
 
-|Azure Devops |Frontend |@backstage/plugin-azure-devops | |0.3.7 |Community Support 
+|Azure Devops |Frontend |@backstage/plugin-azure-devops | |0.3.12 |Community Support 
 |./dynamic-plugins/dist/backstage-plugin-azure-devops |
 |Disabled
 
 |Azure Devops |Backend |@backstage/plugin-azure-devops-backend |Azure
 DevOps backend plugin that contains the API for retrieving builds, pull
-requests, etc. which is used by the Azure DevOps frontend plugin. |0.4.3
-|Tech Preview
+requests, etc. which is used by the Azure DevOps frontend plugin. |0.5.5
+|Community Support
```

Comment:
this was already in the Community support table on release-notes/con-relnotes-techpreview-features.adoc and it's third party owned (not RH) so it should be community according to Christophe.